src/Controller/NetsEasyController.php

```diff
@@ -54,29 +54,30 @@ public function createPayment(Request $request): Response {
     // Amount to pay is defined in the lowest monetary unit.
     $amountToPay *= 100;
     $callbackUrl = $request->get('callbackUrl');
+    $paymentMethods = $request->get('paymentMethods');
+    $paymentMethodsDecoded = NULL;
+    if ($paymentMethods) {
+      parse_str($paymentMethods, $paymentMethodsDecoded);
+    }
```

Comment:
This seems fishy. Pass the list of payment methods as – wait for it – a list in the query (`paymentMethods[]=…&paymentMethods[]=…`).